test/common/upstream/cluster_manager_impl_test.cc

```diff
@@ -568,20 +578,21 @@ TEST_F(ClusterManagerImplTest, RingHashLoadBalancerInitialization) {
     connect_timeout: 0.250s
     type: STATIC
     load_assignment:
+      cluster_name: redis_cluster
       endpoints:
-        - lb_endpoints:
-          - endpoint:
-              address:
-                socket_address:
-                  address: 127.0.0.1
-                  port_value: 8000
-          - endpoint:
-              address:
-                socket_address:
-                  address: 127.0.0.1
-                  port_value: 8001
+      - lb_endpoints:
+        - endpoint:
+            address:
+              socket_address:
+                address: 127.0.0.1
+                port_value: 8000
+        - endpoint:
+            address:
+              socket_address:
+                address: 127.0.0.1
+                port_value: 8001
   )EOF";
-  create(parseBootstrapFromV2Yaml(yaml));
+  create(parseBootstrapFromV3Yaml(yaml));
```

Comment:
Have you update `parseBootstrapFromV3Yaml` to force v3?

Reply:
@htuch yes. code changes is done as per comments.